hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/OMRequestTestUtils.java

```diff
@@ -1623,4 +1625,55 @@ public static OMRequest createRequestWithS3Credentials(String accessId,
         .build();
   }
 
+  /**
+   * Add key entry to PrefixTable.
+   * @throws Exception
+   */
+  public static void addPrefixToTable(String volumeName, String bucketName, String prefixName, long trxnLogIndex,
+      OMMetadataManager omMetadataManager) throws Exception {
+
+    OmPrefixInfo omPrefixInfo = createOmPrefixInfo(volumeName, bucketName,
+        prefixName, trxnLogIndex);
+
+    addPrefixToTable(false, omPrefixInfo, trxnLogIndex,
+        omMetadataManager);
+  }
+
+  /**
+   * Add key entry to PrefixTable.
+   * @throws Exception
+   */
+  public static void addPrefixToTable(boolean addToCache, OmPrefixInfo omPrefixInfo, long trxnLogIndex,
+      OMMetadataManager omMetadataManager) throws Exception {
+    String prefixName = omPrefixInfo.getName();
+
+    if (addToCache) {
+      omMetadataManager.getPrefixTable()
+          .addCacheEntry(new CacheKey<>(omPrefixInfo.getName()),
+              new CacheValue<>(Optional.of(omPrefixInfo), trxnLogIndex));
```

Comment:
Thanks for the review. Updated. 

I also find some in my previous patches (`S3ExpiredMultipartUploadsAbortRequest`). Maybe since `CacheValue` is an internal class we can instead remove it entirely?

Reply:
> Maybe since `CacheValue` is an internal class we can instead remove it entirely?

Yes, the plan is to remove it eventually (as part of HDDS-9985).

Reply:
I see, thank you for the info.